src/UI/Conversation.jsx

```diff
@@ -321,10 +334,14 @@ class Conversation extends Component {
 					onScroll={this.handleScroll}
 					ref={this.bodyRef}
 					role={bodyRole}
-					tabIndex={-1} // tabIndex is required for onKeyDown to work
+					tabIndex={0} // tabIndex is required for onKeyDown to work
 				>
 					{
-						this.state.messages.map((message, index, array) => (
+						this.state.messages.length === 0
+						? <VisuallyHidden>
+							<article>{this.context.noMessagesInConversation}</article>
+						</VisuallyHidden>
+						: this.state.messages.map((message, index, array) => (
 							<React.Fragment key={message.id}>
 								{
 									this.setRenderedDate(this.getDateFromTimestamp(message.time))
```

Comment:
Moet je hier dan niet ook ergens `<article>`  gebruiken?

Reply:
The accessibility reporting does not trigger on this. I think only "interactive" elements need to be inside an `<article>`. So as long as there are no buttons/links etc in there we should be fine.

Reply:
But you have carousel/message underneath, i guess in case of message you don't have buttons but with carousel you would. Also i thought i meant article is just one group of context, and a message could be one group of context